website/docs/r/container_cluster.html.markdown

```diff
@@ -13,9 +13,20 @@ Creates a GKE cluster. For more information see
 and
 [API](https://cloud.google.com/container-engine/reference/rest/v1/projects.zones.clusters).
 
-!> **Warning:** Due to limitations of the API, all arguments except
-`node_version` are non-updateable. Changing any will cause recreation of the
-whole cluster!
+!> **Warning:** Due to limitations of the API, the following arguments are non-updateable:
```

Comment:
Honestly I'm not sure this warning is particularly helpful, and we don't bother showing a similar one for most resources. Do you think it's ok to remove overall or do you think it's better to leave it in?